### PR TITLE
Fix/1948 phpcs

### PR DESCRIPTION
**Pull request by felixarntz**
## Summary

Addresses issue #1948

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

**Review by aaemnnosttv — COMMENTED** [review]
Looks good @felixarntz – just a few suggestions, but nothing blocking.

I confirmed that a missing text-domain and loose equality checks raise errors as expected now 👍

File: tests/phpunit/integration/Core/Storage/Data_EncryptionTest.php [hunk 1]
```diff
@@ -27,8 +27,7 @@ public function test_encrypt() {
 
 		// The result is base64_encoded.
 		$encrypted       = $encryption->encrypt( 'test-value' );
-		$base_64_decoded = base64_decode( $encrypted, true );
-		$this->assertNotFalse( $base_64_decoded );
+		$base_64_decoded = base64_decode( $encrypted, true ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_decode
```
[the remark this answers is not in the document]
Comment (felixarntz) [reply tests/phpunit/integration/Core/Storage/Data_EncryptionTest.php:30]: Good point. I'd say we should rather exclude `WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_decode` and `WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_encode` specifically though, and decide for each of such functions on a case-by-case basis.

For the base64 functions, I'd say it's safe to say we won't use them to "obfuscate code" which is why they're apparently discouraged, so we're good there.

File: tests/phpunit/includes/MutableInput.php [hunk 2]
```diff
@@ -24,7 +24,7 @@ public function filter( $type, $variable_name, $filter = FILTER_DEFAULT, $option
 			INPUT_GET    => $_GET,
 			INPUT_POST   => $_POST,
 			INPUT_SERVER => $_SERVER,
-			INPUT_COOKIE => $_COOKIE,
+			INPUT_COOKIE => $_COOKIE, // phpcs:ignore WordPressVIPMinimum.Variables.RestrictedVariables.cache_constraints___COOKIE
```
[the remark this answers is not in the document]
Comment (felixarntz) [reply tests/phpunit/includes/MutableInput.php:27]: Not sure how you mean? The other references here don't cause any warnings or errors, so I don't think anything else is required.

File: tests/phpunit/includes/Core/Storage/Base64_Encryption.php [hunk 3]
```diff
@@ -26,7 +26,7 @@ class Base64_Encryption {
 	 * @return string
 	 */
 	public function encrypt( $value ) {
-		return base64_encode( $value );
+		return base64_encode( $value ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_encode
```
[the remark this answers is not in the document]
Comment (felixarntz) [reply tests/phpunit/includes/Core/Storage/Base64_Encryption.php:29]: See above, I think for these two functions we can generally exclude.

File: tests/phpunit/includes/MutableInput.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (aaemnnosttv) [reply tests/phpunit/includes/MutableInput.php:27]: My thinking was that if it's flagging the usage of `$_COOKIE` this should be allowed in tests. It's not quite the same error as the other super global use as this warning is specific to cache concerns, so I suppose that's fine. It's not a big deal as we don't use it anyways.

File: tests/phpunit/includes/Core/Storage/Base64_Encryption.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (aaemnnosttv) [reply tests/phpunit/includes/Core/Storage/Base64_Encryption.php]: These exceptions were missed in https://github.com/google/site-kit-wp/pull/1950/commits/742850a34f62549f4c2bcf3931e4e86f353599a4 so I've removed them 👍

File: tests/phpunit/integration/Core/Storage/Data_EncryptionTest.php [hunk 4]
```diff
@@ -28,7 +28,6 @@ public function test_encrypt() {
 		// The result is base64_encoded.
 		$encrypted       = $encryption->encrypt( 'test-value' );
 		$base_64_decoded = base64_decode( $encrypted, true );
-		$this->assertNotFalse( $base_64_decoded );
```
Comment (aaemnnosttv) [inline tests/phpunit/integration/Core/Storage/Data_EncryptionTest.php]: This assertion shouldn't have been deleted. Looking back at the commit, I think it was accidentally included with the `phpcs:ignore` on the line above. I'll restore it 👍

**Review by aaemnnosttv — COMMENTED** [review]
There were just a few minor things that were missed that I fixed, so this should be good to go now.

**Comment by aaemnnosttv** [comment]
@googlebot I consent.

**Review by aaemnnosttv — APPROVED** [review]

**Pull request merged**